src/components/list_group/list_group.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const EuiListGroup = ({
+  children,
+  className,
+  flush,
+  bordered,
+  ...rest
+}) => {
+
+  const classes = classNames(
+    'euiListGroup',
+    {
+      'euiListGroup--flush': flush,
+      'euiListGroup--bordered': bordered,
+    },
+    className
+  );
+
+  return (
+    <ul
+      className={classes}
+      {...rest}
+    >
+      {children}
+    </ul>
+  );
+};
+
+EuiListGroup.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+
+  /**
+   * Remove container padding, stretching list items to the edges
+   */
+  flush: PropTypes.bool.isRequired,
+
+  /**
+   * Add a border to the list container
+   */
+  bordered: PropTypes.bool.isRequired,
```

Comment:
```suggestion
  bordered: PropTypes.bool,
```

Because it's a boolean, it's technically not required since not passing the prop makes it false.